src/main/web_src/src/components/ui/form/inputs/checbox/Checkbox.js

```diff
@@ -5,10 +5,29 @@ import { FormikField } from '~/components/ui/form/FormikField'
 import { SyntEvent } from '~/components/ui/form/formUtils'
 
 import './Checkbox.less'
+import styled from 'styled-components'
 
-export const Checkbox = ({ id, ...restProps }) => {
-	return <NavCheckbox id={id || restProps.name} className="dolly-checkbox" {...restProps} />
-}
+const StyledAttributeCheckbox = styled(NavCheckbox)`
+	&&& {
```

Comment:
Hva gjør &&& ? Apender?

Reply:
Ja appender generert styled navn, noen av de nye nav komponentene har blitt mye mer spesifikke, så må overgå de igjen når vi skal override 😅  Så den blir .styledNavn.styledNavn. osv.. 
Fant det på styled component dokumentasjonen: 

> Finally, the ampersand can be used to increase the specificity of rules on the component; this can be useful if you are dealing with a mixed styled-components and vanilla CSS environment where there might be conflicting styles: